<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotteryClass</class>
 <widget class="QWidget" name="LotteryClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>抽奖</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,2,0,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>微软雅黑</family>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; color:#ff0000;&quot;&gt;常州市北郊初级中学新春联欢会&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="numLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="tipLabel">
     <property name="text">
      <string>单击Space开始</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lottery.qrc"/>
 </resources>
 <connections/>
</ui>
